app/src/main/java/com/example/pokemonapp/ui/PokemonListFragment.kt

```diff
@@ -9,8 +9,10 @@ import androidx.fragment.app.viewModels
 import com.example.pokemonapp.R
 import com.example.pokemonapp.databinding.FragmentPokemonListBinding
 import com.example.pokemonapp.ui.adapters.PokemonItemAdapter
+import dagger.hilt.android.AndroidEntryPoint
 
 
+@AndroidEntryPoint
```

Comment:
Can you explain this

Reply:
it allows  to inject dependencies into the ViewModel without the need to manually create a ViewModelFactory or pass dependencies through the constructor.